### PR TITLE
Initial manifest for com.github.ryanakca.slingshot

### PR DESCRIPTION
**Pull request by cosimoc**

### CODE REVIEW

File: slingshot.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Filed on Fedora bugzilla and mailed contact. Website seems for sale though. -->
+<application>
```
Comment (TingPing) [inline slingshot.appdata.xml]: Use `appstream-util upgrade` for latest format.

File: slingshot.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    </description>
+    <url type="homepage">http://slingshot.wikispot.org/Front_Page</url>
+    <screenshots>
+        <screenshot type="default">http://slingshot.wikispot.org/Screenshots?sendfile=true&amp;file=sshot1.png</screenshot>
+        <screenshot>http://slingshot.wikispot.org/Screenshots?sendfile=true&amp;file=sshot2.png</screenshot>
+        <screenshot>http://slingshot.wikispot.org/Screenshots?sendfile=true&amp;file=sshot3.png</screenshot>
+        <screenshot>http://slingshot.wikispot.org/Screenshots?sendfile=true&amp;file=sshot4.png</screenshot>
+    </screenshots>
+    <updatecontact>jon AT slingshot-game DOT org</updatecontact>
+</application>
```
Comment (TingPing) [inline slingshot.appdata.xml]: Missing OARS.

File: slingshot.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        <p>
+            In order to hit your opponent, you can rotate you ship to aim in any direction, and you can change the power with which your projectile is launched - the initial speed at which it flies.
+        </p>
+        <p>
+            Slingshot is free software as defined by the [WWW]Free Software Foundation, it is released under the [WWW]GNU General Public License. This means you are free to download, copy, modify, redistribute, disassemble, analyse or sell Slingshot under the terms of the aforementioned license. Enjoy!
+        </p>
+    </description>
+    <url type="homepage">http://slingshot.wikispot.org/Front_Page</url>
+    <screenshots>
+        <screenshot type="default">http://slingshot.wikispot.org/Screenshots?sendfile=true&amp;file=sshot1.png</screenshot>
```
Comment (TingPing) [inline slingshot.appdata.xml]: Broken URLs.

File: slingshot.appdata.xml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Filed on Fedora bugzilla and mailed contact. Website seems for sale though. -->
+<application>
+    <id type="desktop">slingshot.desktop</id>
+    <licence>CC0</licence>
```
Comment (TingPing) [inline slingshot.appdata.xml]: This should be `metadata_license` and `CC0-1.0`

File: slingshot.appdata.xml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    <description>
```
Comment (TingPing) [inline slingshot.appdata.xml]: Missing `project_license`

File: slingshot.appdata.xml (hunk 1 above)
Comment (TingPing) [inline slingshot.appdata.xml]: Missing `releases`

**Comment by cosimoc** [comment]
Updated the appdata file to address these concerns, and opened a PR upstream to have it there.

**Comment by TingPing** [comment]
bot, build com.github.ryanakca.slingshot

**Comment by cosimoc** [comment]
Seems to have built fine: https://flathub.org/builds/#/builders/2/builds/3118

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.ryanakca.slingshot

**Pull request closed without merging**